packages/language-tools/language-server/src/plugins/astro.ts

```diff
@@ -14,14 +12,11 @@ import {
 	Range,
 	TextEdit,
 } from '@volar/language-server';
-import fg from 'fast-glob';
-import type { Provide } from 'volar-service-typescript';
 import type { TextDocument } from 'vscode-html-languageservice';
 import { URI } from 'vscode-uri';
 import { AstroVirtualCode } from '../core/index.js';
-import { isJSDocument } from './utils.js';
 
-export const create = (ts: typeof import('typescript')): LanguageServicePlugin => {
+export const create = (): LanguageServicePlugin => {
 	return {
 		capabilities: {
 			completionProvider: {
```

Comment:
I can't add a suggestion for it, but you also need to remove codeLensProvider from the capabilities of this plugin!